app/graphql/mutations/cinema_hall_mutation.rb

```diff
@@ -0,0 +1,12 @@
+module Mutations
+    class CinemaHallMutation < BaseMutation
+      argument :volume, Integer, required: false
+      
+      def resolve(**args)
+        repository = Repository::CinemaHallRepository.new
+        UseCase::CinemaHalls::Create.new(repository).call(args)
+        {status: 200}
```

Comment:
Consider using Rubocop, here you have missing spaces

Reply:
Also (I'm not sure about this one cause last time I've used GraphQL years ago), you don't want to return created object in response?

Reply:
@wmaciejak I'm gonna add https://github.com/testdouble/standard or use rubocop once I'm done with it all. Gonna do it once I finish task from logic side.

Reply:
Okay 👍 Just remember for the future that it's better to standardize code before you start coding (or at least before code review).